custom_components/anova_nano/sensor.py

```diff
@@ -1,59 +1,79 @@
 """Sensor platform for Anova Nano."""
 from __future__ import annotations
 
+from dataclasses import dataclass
+
 from homeassistant.components.sensor import (
+    SensorDeviceClass,
     SensorEntity,
     SensorEntityDescription,
-    SensorDeviceClass,
     SensorStateClass,
 )
+from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import UnitOfTemperature
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
 
 from .const import DOMAIN
 from .coordinator import AnovaNanoDataUpdateCoordinator
 from .entity import AnovaNanoDescriptionEntity
 
-ENTITY_DESCRIPTIONS = (
-    SensorEntityDescription(
-        key="water_temperature",
+
+@dataclass(frozen=True)
+class AnovaSensorEntityDescription(SensorEntityDescription):
+    """Describes Anova sensor entity."""
+
+
+SENSOR_DESCRIPTIONS: tuple[AnovaSensorEntityDescription, ...] = (
+    AnovaSensorEntityDescription(
+        key="water_temp",
+        name="water temperature",
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
         device_class=SensorDeviceClass.TEMPERATURE,
         state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=2,
+        icon="mdi:thermometer",
     ),
-    SensorEntityDescription(
-        key="heater_temperature",
+    AnovaSensorEntityDescription(
+        key="heater_temp",
+        name="heater temperature",
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
         device_class=SensorDeviceClass.TEMPERATURE,
         state_class=SensorStateClass.MEASUREMENT,
     ),
-    SensorEntityDescription(
-        key="triac_temperature",
+    AnovaSensorEntityDescription(
+        key="triac_temp",
+        name="triac temperature",
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
         device_class=SensorDeviceClass.TEMPERATURE,
         state_class=SensorStateClass.MEASUREMENT,
     ),
-    SensorEntityDescription(
-        key="internal_temperature",
+    AnovaSensorEntityDescription(
+        key="internal_temp",
+        name="internal temperature",
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
         device_class=SensorDeviceClass.TEMPERATURE,
         state_class=SensorStateClass.MEASUREMENT,
     ),
-    SensorEntityDescription(
+    AnovaSensorEntityDescription(
         key="motor_speed",
+        name="motor speed",
```

Comment:
I wonder what the unit is for the motor speed. When heating the value is around 160 - perhaps rpm?